packages/ui/src/ui/store/actions/acl.ts

```diff
@@ -59,20 +57,10 @@ function getPathToCheckPermissions(
         case IdmObjectType.ACCOUNT:
             return `//sys/accounts/${entityName}`;
         case IdmObjectType.POOL: {
-            const pools = getPools(state);
-            if (pools?.length) {
-                const poolPath = computePathItems(pools, entityName).filter(
-                    (item) => item !== ROOT_POOL_NAME,
-                );
-                return `//sys/pool_trees/${poolTree}/${poolPath.join('/')}`;
-            } else {
-                const params = new URLSearchParams(window.location.search);
-                const path = params.get('path');
-                if (!path) {
-                    break;
-                }
-                return path;
-            }
+            const fullPath: string = await ytApiV3Id.get(YTApiId.schedulingPoolFullPath, {
```

Comment:
error handling?

Reply:
It is by design, the code should throw error.